doc/src/sgml/parallel.sgml

```diff
@@ -364,43 +555,69 @@ EXPLAIN SELECT * FROM pgbench_accounts WHERE filler LIKE '%x%';
     be performed while parallel query is in use, not even in the leader.
     When a query contains anything which is parallel unsafe, parallel query
     is completely disabled for that query.
+-->
+プランナは、クエリ中に実行される操作を<firstterm>パラレル安全（訳注： parallel safe）</>、<firstterm>パラレル制限（訳注：parallel restricted）</>、<firstterm>パラレル非安全（訳注：parallel unsafe）</>に分類します。
+パラレル制限操作は、パラレルクエリを利用中に、パラレルワーカー中では実行できないが、リーダーによって実行できる操作です。
+したがって、パラレル制限操作は、<literal>Gather</>ノードより下では決して実行されませんが、<literal>Gather</>ノードを含むプランの別の場所では実行されるかもしれません。
+パラレル非安全操作は、パラレルクエリ利用中に、リーダも含めて実行できない操作です。
+クエリがパラレル非安全なものを含む場合は、クエリ中でのパラレルクエリの利用は完全に禁止されます。
   </para>
 
   <para>
+<!--
     The following operations are always parallel restricted.
+-->
+次の操作は常にパラレル制限です。
   </para>
 
   <itemizedlist>
     <listitem>
       <para>
+<!--
         Scans of common table expressions (CTEs).
+-->
+共通テーブル式（CTE）のスキャン
       </para>
     </listitem>
 
     <listitem>
       <para>
+<!--
         Scans of temporary tables.
+-->
+一時テーブルのスキャン
       </para>
     </listitem>
 
     <listitem>
       <para>
+<!--
         Scans of foreign tables, unless the foreign data wrapper has
         an <literal>IsForeignScanParallelSafe</> API which indicates otherwise.
+-->
+外部テーブルのスキャン。
+外部データラッパが<literal>IsForeignScanParallelSafe</>パラレル安全を示しているAPIを持つ場合を除く。
```

Comment:
「APIがパラレル安全を返す場合を除く」は「APIが存在しない」OR「APIが存在してパラレル安全を返さない（つまりfalseを返す）」と同等だと思うのですが…
ただ、誤解を招くのでなければ、細かい表現にこだわるつもりはないので、訳者の選択を尊重します。むしろ、英文と同等でない訳(otherwiseの意味）について確認するために再度コメントしたものなので。